lib/vbridge/VAnchor.ts

```diff
@@ -553,9 +555,12 @@ class VAnchor {
       input.extDataHash.toString()
     );
     //console.log(`current root (class) is ${toFixedHex(this.tree.root())}`);
-    outputs.forEach((x) => {
+    outputs.forEach(async (x) => {
```

Comment:
What's async inside this loop? Also there's nuance using async with loops that don't work with for each afaik. If you need to you should use .map and wrap it in a await Promise.all(array.map(async (a) ...))

Reply:
Yeah not sure, I'm not sure why I put async there. I will take it out.